hr_payroll_commission/models/hr_contract.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# © 2016 Coninckx David (Open Net Sarl)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from openerp import models, fields, api
+import openerp.addons.decimal_precision as dp
+
+
+class HrContract(models.Model):
+    _inherit = 'hr.contract'
+
+    # ---------- Fields management
+    commission = fields.Float(string='Commission',
+                              compute='_comp_commission')
+    comm_rate = fields.Float(string='Commissions Rate',
+                             digits=dp.get_precision('Payroll Rate'))
+
+    @api.multi
+    @api.depends('employee_id.user_id')
+    def _comp_commission(self):
+        for contract in self:
+            contract.commission = 0
```

Comment:
What the point of setting `contract.commission` to 0 to add it `commission` value later?

keeping only the line ``contract.commission = commission` should be enough